spec/models/ability_spec.rb

```diff
@@ -139,6 +140,21 @@
   let(:non_contributor_ephemera_folder) { FactoryBot.create_for_repository(:ephemera_folder) }
   let(:non_contributor_ephemera_box) { FactoryBot.create_for_repository(:ephemera_box, member_ids: contributor_ephemera_folder_in_box.id) }
   let(:non_contributor_ephemera_folder_in_box) { FactoryBot.create_for_repository(:ephemera_folder) }
+  let(:restricted_viewer_collection) do
+    FactoryBot.create_for_repository(:collection, restricted_viewers: [current_user&.uid || "rando"])
```

Comment:
Why this `or`?

Reply:
There's no current_user for the anonymous tests, and it's important for the testing that `restricted_viewers` not be empty.

Reply:
I see. Could you add this as a comment; it doesn't look like it was a pre-existing convention in this spec.

Reply:
The CDL items do it, but yeah I'm happy to add a comment.